en/services/gimbal.md

```diff
@@ -20,3 +22,25 @@ To reboot or shut down a gimbal send the command [MAV_CMD_PREFLIGHT_REBOOT_SHUTD
 
 A gimbal (or mount) should send a [HEARTBEAT](../messages/common.md#HEARTBEAT) (e.g. every second) just like any other MAVLink component. Additionally, it can send feedback about the angles it's pointing using the message [MOUNT_ORIENTATION](../messages/common.md#MOUNT_ORIENTATION).
 
+## Known Issues {#known_issues}
```

Comment:
FYI @julianoes Decided to put this here. I figure worse case it is useful for people to understand the issues they will run into during development.

Reply:
Great, thanks!